client/src/components/Sub_Questions/AnswerEntry.jsx

```diff
@@ -27,6 +27,16 @@ export default function AnswerEntry({ answer, updateAnswers }) {
     <div>
       <div className="answer-entry">
         <span className="answer-item">A:</span> {answer.body}
+        <div className="images">
+          {answer.photos.map((photo, ind) => (
+            <Image
+              key={ind}
+              style={{ width: 100 }}
+              cloudName="dfxarumgq"
```

Comment:
can you explain this cloudName attribute?

Reply:
oh it is like API related thing like our github API key

Reply:
Image is like a component from the library and takes in that prop